src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -146,9 +150,33 @@ public static EventName parseEventName(String name) throws ParseException {
      */
     public static Time parseTime(String time) throws ParseException {
         requireNonNull(time);
-        String trimmedTime = time.trim();
-        //Add validation logic
-        return new Time(trimmedTime);
+
+        String[] trimmedTime = time.trim().split(", ", 2);
+
+        LocalDateTime fromDate;
+        LocalDateTime toDate;
+
+        try {
+            String from = trimmedTime[0].replaceFirst("from: ", "");
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
```

Comment:
Would it help if we declare this as a static formatter in the class and call it rather than instantiate everytime?

Reply:
Thanks! I have implemented the change.